### PR TITLE
New Office tower

### PR DESCRIPTION
**Pull request by John-Candlebury**
A new variation of the office tower for larger city sizes (6 and above).

### CODE REVIEW

File: data/json/overmap_specials.json [hunk 1]
```diff
@@ -534,6 +534,26 @@
         "flags" : [ "CLASSIC" ]
     },{
         "type" : "overmap_special",
+        "id" : "Office Tower 2.0",
```
Comment (KA101) [inline data/json/overmap_specials.json]: OK, I see that it worked OK in the original tower, but we'd like to keep spaces out of id strings where possible.

File: data/json/overmap_specials.json [hunk 2]
```diff
@@ -534,6 +534,26 @@
...
+            { "point":[-1,0,0], "overmap": "office_tower_2_a2_north"},
+            { "point":[0,0,0], "overmap": "office_tower_2_a3_north", "connect": "road"},
+            { "point":[-2,1,0], "overmap": "office_tower_2_b1_north"},
+            { "point":[-1,1,0], "overmap": "office_tower_2_b2_north"},
+            { "point":[0,1,0], "overmap": "office_tower_2_b3_north"}
+        ],
+        "locations" : [ "wilderness" ], "//":"what special locations does it spawn",
+        "city_distance" : [-1, 4], "//":"how far from a city it should be",
+        "city_sizes" : [6, 12], "//":"what city sizes should it spawn in",
+        "occurrences" : [1, 5], "//":"how many per overmap",
```
Comment (KA101) [inline data/json/overmap_specials.json]: oops, this causes the special-crowding--min 1 is similar to required = true.  0-5 would be better.

**Comment by KA101** [comment]
Looks nifty.  I think it's OK to merge this as there's no content freeze just yet, so if you'd fix those I'll get this in.

**Comment by vache** [comment]
I think at the moment, specials should in general only be required if they have a quest attached to them.

**Comment by John-Candlebury** [comment]
Yeah, not a bad policy I would think

**Comment by KA101** [comment]
I approve.

**Comment by Zireael07** [comment]
Cool!

**Pull request merged**